superset/assets/src/dashboard/util/getPublishSubscriberMap.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import * as _ from 'lodash';
+const APPLY_FILTER = 'APPLY_FILTER';
+
+function createPublishData(chart, charts) {
+    return {
+        id: chart.id,
+        publish_columns: chart.formData.publish_columns,
+        subcribers: getSubsribersFor(chart.id, charts),
+        viz_type: chart.formData.viz_type
+    }
+
+}
+
+function getSubsribersFor(publisherId, charts) {
```

Comment:
we are retuning subscriber slices id's. It is not much clear from function name.